app/listeners/webhook_listener.rb

```diff
@@ -51,10 +51,23 @@ def webwidget_triggered(event)
     deliver_webhook_payloads(payload, inbox)
   end
 
+  def contact_created(event)
+    contact = extract_contact_and_account(event)[0]
+    payload = contact.webhook_data.merge(event: __method__.to_s)
+    deliver_account_webhooks(payload, contact.account)
+  end
+
+  def contact_updated(event)
+    contact = extract_contact_and_account(event)[0]
+    changed_attributes = extract_changed_attributes(event)
+    payload = contact.webhook_data.merge(event: __method__.to_s, changed_attributes: changed_attributes)
+    deliver_account_webhooks(payload, contact.account)
+  end
+
   private
 
-  def deliver_account_webhooks(payload, inbox)
-    inbox.account.webhooks.account_type.each do |webhook|
+  def deliver_account_webhooks(payload, account)
+    account.webhooks.account_type.each do |webhook|
```

Comment:
Please change line number 85 as well, as it uses the same method, deliver_account_webhooks. Could you update it to use inbox.account instead of inbox?

Reply:
I've changed it to `account.inbox`  thanks